Runtime/src/index.ts

```diff
@@ -226,6 +226,29 @@ export class SwiftRuntime {
                 this.memory
             );
         },
+        swjs_call_function_unsafe: (
+            ref: ref,
+            argv: pointer,
+            argc: number,
+            kind_ptr: pointer,
+            payload1_ptr: pointer,
+            payload2_ptr: pointer
+        ) => {
+            const func = this.memory.getObject(ref);
+            const result = Reflect.apply(
+                func,
+                undefined,
+                JSValue.decodeArray(argv, argc, this.memory)
+            );
+            JSValue.write(
+                result,
+                kind_ptr,
+                payload1_ptr,
+                payload2_ptr,
+                false,
+                this.memory
+            );
```

Comment:
This will tell the Wasm side if there was an unexpected JS-side exception

```suggestion
            let isException = true;
            try {
                const result = Reflect.apply(
                    func,
                    undefined,
                    JSValue.decodeArray(argv, argc, this.memory)
                );
                isException = false;
                JSValue.write(
                    result,
                    kind_ptr,
                    payload1_ptr,
                    payload2_ptr,
                    false,
                    this.memory
                );
            } finally {
                if (isException) {
                    JSValue.write(
                        undefined,
                        kind_ptr,
                        payload1_ptr,
                        payload2_ptr,
                        true,
                        this.memory
                    );
                }
            }
```

Reply:
Thanks @j-f1 I'm going to update the other function I created with a similar approach as well!

Reply:
"an unexpected JS-side exception" means exceptions thrown by our runtime, right?

Reply:
@kateinoigakukun Not quite! This is for when the user calls a function on the JS side that throws an exception:

```swift
JSObject.global.atob.function!(7)
```

This change will allow the exception to bubble up to the devtools console (along with halting Swift execution), making it easier to debug.